plugins/woocommerce/tests/api-core-tests/playwright.config.js

```diff
@@ -8,10 +8,10 @@ const {
 } = process.env;
 require( 'dotenv' ).config();
```

Comment:
Suggested change--we can update this line to pull in the `.env` file from this directory (`api-core-tests`) instead so we don't need to put the `.env` file up in the `plugins/woocommerce` directory. This also has the added benefit of getting us ready for when we go to publish this a standalone package that is configurable:
```suggestion
require( 'dotenv' ).config({ path: __dirname + '.env' });
```

Reply:
Love this suggestion! Thanks @zhongruige !

Reply:
I like the use of the __dirname (wasn't aware that was a thing!) - small update to get it operational would be a forward slash before the .env i.e.:
require( 'dotenv' ).config({ path: __dirname + `'/.env'` });